ai/toggle_role.py

```diff
@@ -3,23 +3,30 @@
 from roles import has_role
 from typing import List
 import logging
+from webhook import get_webhook_for_channel
+from bot_utils import get_id
 
 LOGGER = logging.getLogger("ai")
 
 
-async def toggle_role(context, targets: List[discord.Member], role_name: str):
+async def toggle_role(context, targets: List[discord.Member], role_name: str, toggle_on: str, toggle_off: str):
```

Comment:
Maybe make the name for these parameters more clear. `toggle_on_response` would be less confusing